source/docs/hardware/sensors/serial-buses.rst

```diff
@@ -43,7 +43,7 @@ The RS-232 bus can also be used through the `MXP expansion port`_.
 
 The roboRIO RS-232 serial port uses RS-232 signaling levels (+/- 15v). The MXP serial port uses CMOS signaling levels with a 3.3v output, and a 3.3v to 5v tolerant input.
 
-.. note:: By default, the onboard RS-232 port is utilized by the roboRIO's serial console. In order to use it for an external device, the serial console must be disabled using the :ref:`Imaging Tool <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO 1>` or :ref:`docs/software/systemcore-info/roborio-web-dashboard:roboRIO Web Dashboard`.
+.. note:: By default, the onboard RS-232 port is utilized by the roboRIO's serial console. In order to use it for an external device, the serial console must be disabled using the //TODO
```

Comment:
Systemcore doesn't have a serial console or RS-232 serial port. This section can be deleted.